pkgs/os-specific/bsd/openbsd/pkgs/libc/package.nix

```diff
@@ -0,0 +1,117 @@
+{
+  lib,
+  stdenv,
+  mkDerivation,
+  bsdSetupHook,
+  openbsdSetupHook,
+  makeMinimal,
+  install,
+  flex,
+  byacc,
+  gencat,
+  rpcgen,
+  lorder,
+  csu,
+  include,
+  ctags,
+  tsort,
+  llvmPackages,
+}:
+
+mkDerivation rec {
+  pname = "libc";
+  path = "lib/libc";
+  extraPaths = [
+    "lib/csu/os-note-elf.h"
+    "sys/arch"
+
+    "lib/libm"
+    "lib/libpthread"
+    "lib/librpcsvc"
+    "lib/librpcsvc"
+    "lib/librthread"
+    "lib/libutil"
+  ];
+
+  patches = [
+    # NetBSD's make usess `${variable:tl}` not `${variable:L}`.
+    ./netbsd-make-to-lower.patch
+    ./libc-no-tags.patch
+    ./disable-librebuild.patch
+    ./libpthread-no-werror.patch
+  ];
+
+  nativeBuildInputs = [
+    bsdSetupHook
+    openbsdSetupHook
+    makeMinimal
+    install
+    flex
+    byacc
+    gencat
+    rpcgen
+    ctags
+    lorder
+    tsort
+  ];
+
+  buildInputs = [
+    include
+    csu
+  ];
+
+  env.NIX_CFLAGS_COMPILE = builtins.toString [ "-B${csu}/lib" ];
```

Comment:
Why make a single element list and then convert it back to a string?

Reply:
There used to be more things in the list :sweat_smile: